app/scripts/web/blueprints/grading.py

```
@@ -0,0 +1,692 @@
+
+    for original_name, student_path in saved:
+        grade_args = _cli() + [
+            "--mode", "grade",
+            "--chunks-jsonl", str(describe_dir / "chunks.jsonl"),
+            "--retrieval-out-jsonl", str(retrieval_out),
+            "--output-root", str(OUTPUT_ROOT),
+            "--run-id", run_id,
+            "--grading-provider", grade_provider,
+            "--grading-model", grade_model,
+            "--student-path", student_path.name,
+        ]
```

Ruff (0.15.11)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)